src/components/Button.js

```diff
@@ -3,23 +3,24 @@ import PropTypes from 'prop-types';
 
 const Button = (props) => {
```

Comment:
there should be a function that calls the clickhandler prop ![screen](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/button/implementHandleClick.png)